Dockerfile

```
@@ -61,19 +77,24 @@ RUN conda clean --all --yes
 ENV RUNNER_CWD=/rmg
 ENV PATH="$RUNNER_CWD/RMG-Py:$PATH"
 
-# 1. Build RMG
-# 2. Install and link Julia dependencies for RMS
+# Build RMG
+RUN make
+
+# Install and link Julia dependencies for RMS
 # setting this env variable fixes an issue with Julia precompilation on Windows
 ENV JULIA_CPU_TARGET="x86-64,haswell,skylake,broadwell,znver1,znver2,znver3,cascadelake,icelake-client,cooperlake,generic"
+ENV RMS_BRANCH=${RMS_Branch}
+# Usually this is set automatically, but we're not actually running
+# in an active conda environment when building the Docker so we need to set it manually
+ENV PYTHON_JULIAPKG_PROJECT="/miniconda/envs/rmg_env/julia_env"
 RUN source install_rms.sh
 
 # RMG-Py should now be installed and ready - trigger precompilation and test run
-RUN python rmg.py examples/rmg/minimal/input.py
-# delete the results, preserve input.py
-RUN mv examples/rmg/minimal/input.py . && \
-    rm -rf examples/rmg/minimal/* && \
-    mv input.py examples/rmg/minimal/
+RUN python rmg.py examples/rmg/rms_constant_V/input.py
+# delete the results, restore input.py from git
+RUN rm -rf examples/rmg/rms_constant_V/* && \
+    git checkout -- examples/rmg/rms_constant_V/
 
 # when running this image, open an interactive bash terminal inside the conda environment
-RUN echo "source activate rmg_env" >~/.bashrc
+RUN echo "conda activate rmg_env" >> ~/.bashrc
```

```suggestion
RUN echo "source activate rmg_env" >> ~/.bashrc
```

We never run `conda init` so we can't run `conda activate`, we have to use `source activate` to get into the environment. I confirmed this locally - changing this line still drops you in a conda environment on running the image, the version in this PR does not.

---

Thanks. Would running `conda init` be a good idea, so that people inside the docker have a conda setup that looks more "normal"?

---

Good idea, added in [c1792e4](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2808/commits/c1792e4aef7441e24e8c8aeeec2a1f9d89d28b8f) so that now `conda activate rmg_env` works as expected